packages/store/src/data-access/filesystem.ts

```diff
@@ -32,20 +28,14 @@ export const FilesystemStore: DataAccessConstructor<object> = class implements D
         this.storeDirectory = basedir(options);
     }
 
-    public async read<Entity extends object>({
-        entityName,
-        id
-    }: {
-        entityName: string;
-        id: string;
-    }): Promise<undefined | Entity> {
+    public async read({ entityName, id }: { entityName: string; id: string }): Promise<undefined | E> {
```

Comment:
Removal of generic, is that expected?

Reply:
Yeah this was showing up as a code smell in sonar.